packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/AccessRequirementTextInstructions.tsx

```diff
@@ -31,34 +32,39 @@ export const AccessRequirementTextInstructions: React.FunctionComponent<
           <Typography variant="body1" mb={1}>
             {oldInstructions}
           </Typography>
-          <Button
-            variant="contained"
-            color="error"
-            onClick={() => setOpen(true)}
-          >
-            Delete old instructions
-          </Button>
-          <ConfirmationDialog
-            open={open}
-            title="Are you sure?"
-            content="Deleting the old instructions cannot be undone. Continue?"
-            onCancel={() => setOpen(false)}
-            onConfirm={() => {
-              const clearedInstructionsAr = { ...accessRequirement }
-              if (
-                clearedInstructionsAr.concreteType ===
-                TERMS_OF_USE_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE
-              ) {
-                clearedInstructionsAr.termsOfUse = undefined
-              } else if (
-                clearedInstructionsAr.concreteType ===
-                ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE
-              ) {
-                clearedInstructionsAr.actContactInfo = undefined
-              }
-              onConfirmDelete(clearedInstructionsAr)
-            }}
-          />
+          {allowDelete && (
```

Comment:
Only change here is wrapping the delete button and confirmation dialog in `{allowDelete && ...`